vendor.conf

```diff
@@ -14,7 +14,7 @@ github.com/cpuguy83/go-md2man                       20f5889cbdc3c73dbd2862796665
 github.com/creack/pty                               2769f65a3a94eb8f876f44a0459d24ae7ad2e488 # v1.1.7
 github.com/davecgh/go-spew                          8991bc29aa16c548c550c7ff78260e27b9ab7c73 # v1.1.1
 github.com/dgrijalva/jwt-go                         a2c85815a77d0f951e33ba4db5ae93629a1530af
-github.com/docker/compose-on-kubernetes             cc4914dfd1b6684a9750a59f3613fc0a95291824 # v0.4.23
+github.com/docker/compose-on-kubernetes             f48765330f6dcda6955f07192b8cc666df8ac580 # v0.4.25-alpha1
```

Comment:
Given the major version bump of kubernetes, we should probably make this `v0.5.0-alpha`, so that we can maintain the `v0.4.x` version to maintain existing releases of the CLI (18.09, 19.03)